release/dashboard/atari.yaml

```diff
@@ -0,0 +1,24 @@
+atari-ppo:
```

Comment:
is this based on some other release test?

Reply:
no. This is the repro script for the memory leak.